.gitignore

```diff
@@ -79,3 +79,4 @@ tags
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
 .idea/
 vendor/
+/log.txt
```

Comment:
Added on purpose?

Reply:
Yes, sometimes I start the operator with a redirection of its output into a log file. Since that log file does not belong into the repo i thought i add it to the gitignore so i don't accidentally commit it.